cli.js

```diff
@@ -31,7 +31,20 @@ var cli = meow({
     '                            Example of a config script: https://github.com/stevemao/conventional-changelog-angular/blob/master/index.js',
     '                            This value is ignored if preset is specified',
     '',
-    '  -c, --context             A filepath of a javascript that is used to define template variables'
+    '  -c, --context             A filepath of a javascript that is used to define template variables',
+    '',
+    'Options ( Github Enterprise )',
+    '  --github-host              Github host name',
```

Comment:
I don't think this should be prefixed with `--github`, this module is all about `github` so no need to mention it in the options.